simtools/util/names.py

```diff
@@ -27,6 +28,81 @@
     "validate_telescope_name_db",
 ]
 
+lst = "LST"
```

Comment:
I think it would be better to read the code like it is now, but I returned it to the way it was to avoid duplication

Reply:
actually changing the dictionary to value: key would disrupt the logic of the variables defined in names. Please have a look at the other all_* variables defined there. using single lst, mst, sct, sst parameters makes the code more readable despite the apparent duplication in names. therefore, I prefer to keep the logic and the readability, but if you prefer you can change it in names. in case you do please make sure you propagate the change to the validation functions

Reply:
You can keep it as you had it originally, no problem. We'll see if Gernot complains =)